Documentation/concepts/overview.rst

```diff
@@ -11,97 +11,81 @@ Component Overview
-
-
-Cilium CLI Client
-=================
-
-The Cilium CLI Client (cilium) is a command-line tool that is installed along
-with the Cilium Agent.  It gives a command-line interface to interact with all
-aspects of the Cilium Agent API.   This includes inspecting Cilium's state
-about each network endpoint (i.e., container), configuring and viewing security
-policies, and configuring network monitoring behavior.
-
-Linux Kernel BPF
-================
-
-Berkeley Packet Filter (BPF) is a Linux kernel bytecode interpreter originally
-introduced to filter network packets, e.g. tcpdump and socket filters. It has
-since been extended with additional data structures such as hashtable and
-arrays as well as additional actions to support packet mangling, forwarding,
-encapsulation, etc. An in-kernel verifier ensures that BPF programs are safe to
-run and a JIT compiler converts the bytecode to CPU architecture specific
-instructions for native execution efficiency. BPF programs can be run at
-various hooking points in the kernel such as for incoming packets, outgoing
-packets, system calls, kprobes, etc.
+A deployment of Cilium and Hubble consists of the following components running
+in a cluster:
+
+
+Cilium
+======
+
+Agent
+  The Cilium agent (``cilium-agent``) runs on each node in the cluster. At a
+  high-level, the agent accepts configuration via Kubernetes or APIs that
+  describes networking, service load-balancing, network policies, and
+  visibility & monitoring requirements.
+
+  The Cilium agent listens for events from orchestration systems such as
+  Kubernetes to learn when containers or workloads are started and stopped. It
+  manages the eBPF programs which the Linux kernel uses to control all network
+  access in / out of those containers.
+
+Client (CLI)
+  The Cilium CLI client (``cilium``) is a command-line tool that is installed
+  along with the Cilium agent. It interacts with the REST API of the Cilium
+  agent running on the same node. The CLI allows inspecting the state and
+  status of the local agent. It also provides tooling to directly access the
+  eBPF maps to validate their state.
+
+Operator
+  The Cilium Operator is responsible for managing duties in the cluster which
+  should logically be handled once for the entire cluster, rather than once for
+  each node in the cluster. The Cilium operator is not in the critical path for
+  any forwarding or network policy decision. A cluster will continue to
+  function if the operator is temporarily unavailable. However, depending on
+  the configuration, it can be involved in :ref:`address_management` and thus
+  delay scheduling of new workloads if unavailable.
+
+CNI Plugin
+  The CNI plugin (``cilium-cni``) is invoked by Kubernetes when a pod is
+  scheduled or terminated on a node. It interacts with the Cilium API of the
+  node to trigger the necessary datapath configuration to provide networking,
+  load-balancing and network policies for the pod.
+
+Hubble
+======
+
+Agent
```

Comment:
kind of feel like we should pick one word, either agent or server, but not both. i'm slightly inclined to call it Hubble Server, but i can go either way.